kolibri/core/assets/src/views/ImmersiveToolbar.vue

```diff
@@ -15,14 +15,15 @@
       v-if="hasRoute"
       slot="icon"
       :to="route"
-      :class="['link', $computedClass(linkStyle)]"
+      class="link"
+      :class="$computedClass(linkStyle)"
     >
       <!-- TODO add aria label? -->
       <UiIconButton
         type="flat"
         class="icon"
         :style="{fill: $themeTokens.textInverted}"
-        @click="$emit('navIconClick')"
+        tabindex="-1"
```

Comment:
There's a listener in CoreBase for `nav-icon-click`. See if that can be removed too